tests/cli/ursula/test_run_ursula.py

```diff
@@ -45,8 +46,7 @@ def test_missing_configuration_file(default_filepath_mock, click_runner):
     result = click_runner.invoke(nucypher_cli, cmd_args, catch_exceptions=False)
     assert result.exit_code != 0
     assert default_filepath_mock.called
-    assert "run: 'nucypher ursula init'" in result.output
-
+    assert "No Ursula configurations found.  run 'nucypher ursula init' then try again." in result.output
```

Comment:
If you use the previous suggestion, this also needs to be updated.
```suggestion
    assert "No Ursula configurations found.  Run 'nucypher ursula init' then try again." in result.output
```

Reply:
:heavy_check_mark: